package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "pretest": "eslint index.js lib test/*.js",
-    "test": "nyc tape test/*.test.js",
-    "coverage": "nyc --reporter html tape test/*.test.js && opener ./coverage/index.html",
+    "test": "nyc mocha test/*.test.js",
```

Comment:
If you want better output put `tap-spec` in as a dev dependency and make the test script pipe into it:

```
"test": "nyc tape test/*.test.js | tap-spec
```

We need to maintain test consistency in our backend systems to maintain human sanity as we move from one project to another and another and back again. There's no doubt that the test suite in cardboard is crummy, but that's not tape's fault. 

You're removing so much code that I almost wonder if the test suite is salvageable. Sometimes these heavy refactors call for a test suite rewrite, where you just recreate the old assertions in a more-sane fashion.